Author: Robert James Clay <jame@rocasa.us>
Description: Set 'lsmb_dbadmin' as the default database superuser in setup.pl.
Origin: vendor
Last-Update: 2016-11-12
--- a/UI/setup/credentials.html
+++ b/UI/setup/credentials.html
@@ -12,7 +12,7 @@
 <div class="tabular col-1">
 <div id="userpass">
 <div class="inputrow">
-<?lsmb IF !s_user; s_user = 'postgres'; END;
+<?lsmb IF !s_user; s_user = 'lsmb_dbadmin'; END;
 INCLUDE input element_data = {
     name = 's_user'
    value = s_user
